gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineTesting.java

```diff
@@ -47,6 +47,9 @@ public void apply(Project project) {
                         .findAny()
                         .ifPresent(ignored -> enableJUnit5ForAllTestTasks(project));
             });
+
+            // Never cache test tasks, until we work out the correct inputs for ETE / integration tests
+            project.getTasks().withType(Test.class).configureEach(test -> test.getOutputs().cacheIf(task -> false));
```

Comment:
There were some requests internally to allow users to opt out of this behaviour. We could expose a property that allows people to toggle ti

Reply:
`com.palantir.baseline.restore-test-cache = true` ?